setup.py

```diff
@@ -2,7 +2,7 @@
 
 from setuptools import setup, find_packages
 
-version = '1.1dev'
+version = '1.0.1'
```

Comment:
actually, wea re preparing tools version 1.1.0, don't we?  So I guess we could leave it like 1.1dev, or 1.1.0 dev.